_data/gallery.yml

```diff
@@ -1,17 +1,23 @@
-- image: /media/gallery/1.jpg
-  title: A tranquil bay with grass and trees
+- image: /media/gallery/mtg.jpg
+  title: A castle in Minetest Game
 
-- image: /media/gallery/2.jpg
-  title: South pacific old growth forest
+- image: /media/gallery/rpg16.jpg
+  title: "[RPG-16](https://content.minetest.net/packages/Hugues%20Ross/rpg16/) texture pack"
 
-- image: /media/gallery/3.jpg
-  title: Castles in a large multiplayer server
+- image: /media/gallery/soothing32.jpg
+  title: "[Soothing32](https://content.minetest.net/packages/Zughy/soothing32/) texture pack"
```

Comment:
PS: not a fan of the restricted palette of this pack, maybe just showcasing one texture pack is enough?

Reply:
> PS: not a fan of the restricted palette of this pack, maybe just showcasing one texture pack is enough?

Personal preferences shouldn't matter here considering that Soothing 32 is the most popular pack on CDB with 20k downloads, significantly more than RPG 16 with 17k downloads (despite the latter having been around longer). Showcasing the top two CDB texture packs is definitely reasonable.